### PR TITLE
Or 1310 audit log rebased

### PR DESCRIPTION
**Pull request by koenmetsu**

### CODE REVIEW

File: src/AssociationRegistry.Admin.Api/Connectionstrings.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+namespace AssociationRegistry.Admin.Api;
+
+using System.Collections.Generic;
+
+public class Connectionstrings : Dictionary<string, PostgreSqlConnectionString>
```
Comment (koenmetsu) [inline src/AssociationRegistry.Admin.Api/Connectionstrings.cs:5]: remove this

File: src/AssociationRegistry.Admin.Api/Startup.cs [hunk 2]
```diff
@@ -48,15 +48,22 @@ public Startup(IConfiguration configuration)
     /// <param name="services">The collection of services to configure the application with.</param>
     public IServiceProvider ConfigureServices(IServiceCollection services)
     {
-        var postgreSqlOptions = _configuration.GetSection("PostgreSQLOptions")
+        var postgreSqlOptionsSection = _configuration.GetSection("PostgreSQLOptions")
```
Comment (koenmetsu) [inline src/AssociationRegistry.Admin.Api/Startup.cs:51]: extract to constant

File: test/AssociationRegistry.Test/Admin.Api.IntegrationTests/Fixtures/AdminApiFixture.cs [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+    protected AdminApiFixture(string identifier)
+    {
+        _identifier += identifier.ToLowerInvariant();
+        _configurationRoot = GetConfiguration();
+    }
+
+    public virtual async Task InitializeAsync()
+    {
+        // await WaitFor.PostGreSQLToBecomeAvailable(
```
Comment (koenmetsu) [inline test/AssociationRegistry.Test/Admin.Api.IntegrationTests/Fixtures/AdminApiFixture.cs:45]: remove comment

File: test/AssociationRegistry.Test/Admin.Api.IntegrationTests/When_posting_a_new_vereniging/Given_A_Request_With_Missing_Name.cs [hunk 4]
```diff
@@ -1,54 +1,56 @@
...
-
-    private string GetJsonRequestBody()
-        => GetType()
-            .GetAssociatedResourceJson($"files.request.empty");
-
-    private string GetJsonResponseBody()
-        => GetType()
-            .GetAssociatedResourceJson($"files.response.not_null_validation_error");
-}
+// namespace AssociationRegistry.Test.Admin.Api.IntegrationTests.When_posting_a_new_vereniging;
```
Comment (koenmetsu) [inline test/AssociationRegistry.Test/Admin.Api.IntegrationTests/When_posting_a_new_vereniging/Given_A_Request_With_Missing_Name.cs:1]: enable test again

File: test/AssociationRegistry.Test/Admin.Api.IntegrationTests/EventStoreTests/When_storing_an_event/Given_An_Event.cs [hunk 5]
```diff
@@ -1,45 +1,57 @@
...
+        var eventStore = new EventStore(_fixture.DocumentStore);
 
         // act
-        await eventStore.Save(
-            streamId,
-            new CommandMetadata(
-                "SomeInitiator",
-                new DateTime(2022, 1, 1).ToUniversalTime().ToInstant()),
-            someEvent);
+        await eventStore.Save(streamId, new CommandMetadata("SomeInitiator", new DateTime(2022, 1, 1).ToUniversalTime().ToInstant()), someEvent);
```
Comment (talboomerik) [inline test/AssociationRegistry.Test/Admin.Api.IntegrationTests/EventStoreTests/When_storing_an_event/Given_An_Event.cs:38]: multi line?

**Comment by koenmetsu** [comment]
check for unused classes

**Pull request merged**